frontend/components/ManageStakingPage/StakingContractSection/CantMigrateAlert.tsx

```diff
@@ -61,24 +59,26 @@ const AlertInsufficientMigrationFunds = ({
 
   if (!isAllStakingContractDetailsRecordLoaded) return null;
   if (isNil(requiredStakedOlas)) return null;
-  if (isNil(safeBalance?.[TokenSymbol.OLAS])) return null;
+  if (isNil(masterSafeBalanceOnHomeChain?.[TokenSymbol.OLAS])) return null;
 
-  if (isNil(totalStakedOlasBalance)) return null;
+  if (isNil(totalStakedOlasBalanceOnHomeChain)) return null;
 
-  const requiredOlasDeposit = isServiceStaked
-    ? requiredStakedOlas -
-      (totalStakedOlasBalance + safeBalance[TokenSymbol.OLAS]) // when staked
-    : requiredStakedOlas - safeBalance[TokenSymbol.OLAS]; // when not staked
+  const requiredOlasDeposit =
+    requiredStakedOlas -
+    sum([
+      masterSafeBalanceOnHomeChain[TokenSymbol.OLAS],
+      totalStakedOlasBalanceOnHomeChain,
+    ]);
```

Comment:
removed this condition, may be breaking due to suspension or eviction
`sum` allows for null/undefined so don't need to guard here
middleware will migrate funds from the current contract if migration required